Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt update && \
 		qt6-wayland-dev qt6-declarative-dev qt6-declarative-private-dev qt6-wayland-private-dev libspng-dev
 
 # enabling gcc 15
-RUN apt -t experimental install -y g++-15
+RUN apt -t experimental install -y g++-15 binutils 
```

Comment:
You said this on the other thread, but why do we need binutils? I didn't quite get this bit.